tests/testJobManager.py

```
@@ -0,0 +1,142 @@
+import unittest
+import redis
+from mock import MagicMock, patch
+
+from jobManager import JobManager
+from jobQueue import JobQueue
+from tangoObjects import TangoJob, TangoMachine
+from config import Config
+from preallocator import *
```

Ruff

`from preallocator import *` used; unable to detect undefined names

(F403)